tests/integration/tier0/single-yum-transaction-validation/main.fmf

```diff
@@ -0,0 +1,33 @@
+summary: Verify single yum transaction validation
+
+description: >
+    Verify that we are doing a proper rollback during the validation phase in
+    our transactions.
+
+    If any errors occurs during the transaction resolution, either by
+    downloading a package, dependency resolver and etc... A rollback should
+    start and revert the changes to the system.
+
+link: https://issues.redhat.com/browse/RHELC-576
+
+tier: 0
+
+tag+:
+    - yum
+    - dnf
+    - transaction
+
+/transaction_validation_error:
+    adjust+:
+        - enabled: false
+          when: distro == centos-8 or distro == oraclelinux-8
```

Comment:
I'd love to add a `because:` here to say why I'm disabling this test for centos-8 and oraclelinux-8, but since it's been a really long time, I don't remember. I will try to get the rationale at some point.